### ISSUE TITLE
Some PDF pages failing to render with `DataCloneError`

### ISSUE THREAD

**Issue by robertknight** [hypothesis/browser-extension#626]
In Chrome 94 I've encountered a number of pages in various PDFs fail to render or render incompletely. In the browser console I see this error:

```
Warning: getOperatorList - ignoring errors during "GetOperatorList: page 0" task:
"DataCloneError: Failed to execute 'postMessage' on 'DedicatedWorkerGlobalScope': 
function at() { [native code] } could not be cloned.".
```

This looks to me like it might relate to the recently introduced `at` method [for strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at) or [arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at).

If so, updating PDF.js to the latest version might fix it.

Example URLs:

https://cogcomp.seas.upenn.edu/papers/RRDA11.pdf
https://files.libertyfund.org/files/869/0161_Bk.pdf

**Comment by robertknight** [comment]
Related upstream issue: https://github.com/mozilla/pdf.js/issues/13669.

From [a comment](https://github.com/mozilla/pdf.js/issues/13669#issuecomment-889062050) in that thread:

> The cause of this error is the, fairly recently added, Array.prototype.at() method which was fixed in PR #12924 and is included in PDF.js release 2.8.335 and up.

**Comment by robertknight** [comment]
This was resolved via the PDF.js update linked above.
